features/obtener_videos_por_visibilidad.feature

```diff
@@ -3,10 +3,12 @@ Feature: Obtener videos por visibilidad
     Background: Estoy logueado y existe el usuario "edson"
         Given inicie sesión correctamente
         And "edson" se registró correctamente
+        And "otro" se registró correctamente
 
     Scenario: Obtener videos privados de un amigo
         Given "edson" es mi amigo
         Given "edson" subio un video privado
+        Given "otro" subio un video publico
```

Comment:
algo asi hay que hacer en listar_videos.feature que es lo que prueba esa feature mas a fondo

Reply:
Debería ser:
```
Given ....
And ...
```
En lugar de meter 3 given

Reply:
ahi agregue un step mas en las pruebas de listar videos y cambie los Given's por And's